src/components/LayerCollection/LayerPOLine.js

```diff
@@ -128,7 +128,7 @@ function LayerPOLine({
   );
 
   const openOrder = useCallback(
-    (saveAndOpen) => {
+    (saveAndOpen, poLineId) => {
```

Comment:
this is not belong here, it's simply not used in this function

Reply:
fixed